docs/source/tutorials/extending-with-plugins/reader-quickguide.rst

```diff
@@ -0,0 +1,221 @@
+Guide to Writing a GeoIPS Reader
+================================
+
+Overview
+--------
+A GeoIPS reader is a module-based plugin that reads data from specific sources (satellite data, environmental data, etc.) and formats it for use within the GeoIPS framework.
+
+Reader Structure
+---------------
+1. **Call Function** (Required)
+   
+   - Main driver that processes input files
+   - Accepts file names and control parameters
+   - Returns dictionary of xarray Datasets
+
+2. **Read Functions** (Recommended)
+   
+   - Handle actual data reading from files
+   - Convert data into xarray format
+
+3. **Utility Functions** (Optional)
+   
+   - Help with data conversion/manipulation
+   - Format time, coordinates, etc.
+
+Required Output Components
+-------------------------
+
+Xarray Variables
+~~~~~~~~~~~~~~
+- ``latitude``, ``longitude`` (REQUIRED): 2D arrays matching data variables
+- ``time`` (OPTIONAL): 2D array for temporal information
```

Comment:
This is probably fine for now since this is what we currently do, but:
1. Is it possible that these might be 1D for some readers?
2. Is it also possible that ``time`` might be 0D for some situations?

Reply:
Yes, and likely `time` would not exist if it's 0d since it is also in the data metadata. Or at least that's what happens now.